Asterisk.2013/Asterisk.NET/Manager/Action/ParkAction.cs

```diff
@@ -50,7 +54,12 @@ public override string Action
         /// <summary>
         ///     Set the Channel where the Call will end up after the timeout is reached.
         /// </summary>
-        public string Channel2 { get; set; }
+        public string TimeoutChannel { get; set; }

```

Comment:
This breaks backward compatibility with Asterisk version <12.
- https://wiki.asterisk.org/wiki/display/AST/Asterisk+11+ManagerAction_Park
- https://wiki.asterisk.org/wiki/display/AST/Asterisk+12+ManagerAction_Park
Maybe we should leave the `Channel2` property and have it simply be a copy of `TimeoutChannel`.
For example:
``` C#
public string Channel2 { get { return TimeoutChannel; } set { TimeoutChanne = valuel; } }
```
I don't have a way to test this though, so if you could test it against both Asterisk versions (<12 and >=12) or something it would be perfect.